Example/StripeConnectExample/StripeConnectExample/MainViewController.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  MainViewController.swift
+//  StripeConnect Example
+//
+//  Created by Mel Ludowise on 4/30/24.
+//
+
+@_spi(PrivateBetaConnect) import StripeConnect
+import SwiftUI
+import UIKit
+
+class MainViewController: UITableViewController {
```

Comment:
Was there a reason you opted to keep this in UIKit instead of rewriting in SwiftUI? There's nothing wrong with keeping it as-is, but if it was in SwiftUI it might save us from needing `SwiftUIContainerViewController`

Reply:
I figured a hybrid SwiftUI/UIKit app is what most people are operating on today. Also the SDK does not offer View equivalents in the API. I thought a UIKit version of this was the best way to show how to use the SDK without all of the added complexity of wrapping the view controllers in views.